src/components/TextInput/BaseTextInput.js

```diff
@@ -129,7 +129,13 @@ class BaseTextInput extends Component {
     onBlur(event) {
         if (this.props.onBlur) { this.props.onBlur(event); }
         this.setState({isFocused: false});
-        this.deactivateLabel();
+
+        // If the text has been supplied by Chrome autofill, the value state is not synced with the value
+        // as Chrome doesn't trigger a change event. When there is autofill text, don't deactivate label.
+        const textWasAutoFilled = this.input.matches && this.input.matches(':-internal-autofill-selected');
```

Comment:
I changed the selector to `-webkit-autofill`.

Reply:
@parasharrajat The original issue doesn't happen on Firefox probably because it fires `change` event. When I tested with `':-webkit-autofill, :autofill'`, the label doesn't deactivate on Chrome after I edit and clear the autofilled text to be blank.